v23.1/transactions.md

```diff
@@ -57,29 +57,28 @@ To handle errors in transactions, you should check for the following types of se
 
 Type | Description
 -----|------------
-**Retry Errors** | Errors with [the code `40001` or string `restart transaction`](common-errors.html#restart-transaction), which indicate that a transaction failed because it could not be placed in a serializable ordering of transactions by CockroachDB. This is often due to [contention](#transaction-contention): conflicts with another concurrent or recent transaction accessing the same data. In such cases, the transaction needs to be retried by the client as described in [client-side retry handling](transaction-retry-error-reference.html#client-side-retry-handling). For a reference listing all of the retry error codes emitted by CockroachDB, see the [Transaction Retry Error Reference](transaction-retry-error-reference.html#transaction-retry-error-reference).
+**Transaction Retry Errors** | Errors with the code `40001` and string `restart transaction`, which indicate that a transaction failed due to [contention](performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention) with another concurrent or recent transaction attempting to write to the same data. For details on transaction retry errors and how to resolve them, see the [Transaction Retry Error Reference](transaction-retry-error-reference.html#actions-to-take).
```

Comment:
> with another concurrent or recent transaction attempting to write to the same data 
^Lock conflicts. We should probably call out serializable conflicts which are the direct cause of retry errors